ZAPD/Globals.cpp

```diff
@@ -66,6 +66,22 @@ std::string Globals::FindSymbolSegRef(int32_t segNumber, uint32_t symbolAddress)
 	return "ERROR";
 }
 
+enum class ConfigType
+{
+	SYMBOL_MAP,
+	SEGMENT,
+	ACTOR_LIST,
+	OBJECT_LIST,
+	TEXTURE_POOL,
+	BG_CONFIG
+};
+
+const static std::map<std::string, ConfigType> ConfigTypeDictionary = {
+	{"SymbolMap", ConfigType::SYMBOL_MAP},     {"Segment", ConfigType::SEGMENT},
+	{"ActorList", ConfigType::ACTOR_LIST},     {"ObjectList", ConfigType::OBJECT_LIST},
+	{"TexturePool", ConfigType::TEXTURE_POOL}, {"BGConfig", ConfigType::BG_CONFIG},
+};
+
 void Globals::ReadConfigFile(const std::string& configFilePath)
 {
 	tinyxml2::XMLDocument doc;
```

Comment:
Changing from an if-else chain to a switch statement here doesn't seem to offer much benefit. You're still left with a chain (one of cases), and with the added enum and dictionary it seems it's gotten bulkier.

Reply:
Addressed in #168 , so I'll revert this.